src/main/resources/application.yaml

```diff
@@ -44,6 +44,11 @@ hearing:
   api:
     url: ${HEARING_API_URL:http://rd-commondata-api-demo.service.core-compute-demo.internal}
 
+
+ref_data:
+  baseUrl: ${REF_DATA_URL:http://rd-location-ref-api-demo.service.core-compute-demo.internal/}
+
```

Comment:
REF_DATA_URL need to set it in values.yml

Reply:
done